h2o-py/requirements.txt

```diff
@@ -24,3 +24,4 @@ mysqlclient
 recommonmark
 sphinx_rtd_theme
 sphinxcontrib-osexample
+fastparquet
```

Comment:
In this specific case I would prefer to have the file statically available on S3. This is good for backwards compatibility tests.

And this is unrelated to the fact that fastparquet has dependencies that can conflict with our own :)

The right thing to do here IMHO is to create the file in Spark/Hive and use put it to S3.

Alternative solution is to write the file in junits.

Reply:
Side note: requirements.txt should be used only for development. Actual package should use deps from `setup.py`.

Reply:
Very good point, thanks @mmalohlava - I did not catch that!

Reply:
Thank you for your comments. I have resolved this issue. As soon as test parquet file is uploaded, I will push the changed requirements.txt and an updated test.